gradle/publishing.gradle

```diff
@@ -14,6 +14,63 @@
  * limitations under the License.
  */
 
+apply plugin: 'java-gradle-plugin'
+apply plugin: 'com.gradle.plugin-publish'
+//apply plugin: 'com.jfrog.bintray'
```

Comment:
I have not enabled Bintray yet, as we need to discuss its purpose.

Reply:
OK, we can keep it out as long as the other publishing methods work